parsl/tests/test_checkpointing/test_regression_232.py

```diff
@@ -0,0 +1,91 @@
+import pickle
+import time
+import subprocess
+import signal
+import os
+
+
+def kill():
+    my_pid = os.getpid()
+    os.kill(my_pid, signal.SIGINT)
+    print("Killing self")
+
+
+def test_regress_232_task_exit(count=2):
+    """Recovering from a run that was SIGINT'ed with task_exit checkpointing
+    """
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    checkpoint_file = os.path.join(cwd, 'checkpointed.py')
+
+    proc = subprocess.Popen("python3 {} -n {} -m task_exit".format(checkpoint_file, count),
```

Comment:
Why not just import `run_checkpointed` and run it directly? Is a subprocess needed?

Reply:
We want this to run as a separate process, because if we import it then we can't SIGINT the dfk.